devtools/prettier.py

```diff
@@ -239,6 +246,26 @@ def _format_dataclass(self, value: 'Any', _: str, indent_current: int, indent_ne
             self._stream.write(',\n')
         self._stream.write(indent_current * self._c + ')')
 
+    def _format_sqlalchemy_class(self, value: 'Any', _: str, indent_current: int, indent_new: int):
+        import json
+
+        fields = {}
+        for field in [x for x in dir(value) if not x.startswith('_') and x != 'metadata']:
+            data = value.__getattribute__(field)
+            try:
+                json.dumps(data)
+                fields[field] = data
+            except TypeError:
+                fields[field] = None
```

Comment:
I don't understand why the formatted value needs to be JSON serialisable. I definitely don't think it should show `foo=None` when that's not the case.

Reply:
I actually took this code from https://stackoverflow.com/a/10664192 this answer.
 json.dumps stands for checking possible encoding errors.

Reply:
But this is a print debugging tool, it doesn't need to encode things as JSON

Reply:
Yes you are right. I removed json parts.